### DISCUSSION TITLE
install for windows

### DISCUSSION THREAD

**Discussion by MansoorRafiq** [zed-industries/zed#19596] [Feature Requests]
Hi, i really like zed code editor because it is very fast and lightweight and it has all the features that every coders need to work faster. The problem is that i' m a window user and i try to build zed editor from source but it fails. I like this editor and i want to use it can you tell me how to use it can zed editor use GPU and i don' t have a GPU.

**Comment by SomeoneToIgnore** [comment]
> how to use it can zed editor use GPU and i don' t have a GPU.

There's no other way, unfortunately.

Currently, Zed uses [`blade`](https://github.com/kvark/blade/) for rendering outside of macOS (so, on Windows it's `blade` too) which imposes relatively high requirements for the GPU features supported.

Alternative backends were not accepted prior: https://github.com/zed-industries/zed/pull/13684
with the intention to move all graphics backend-related code either to `blade` or to a better successor library, but avoid dealing with it in Zed.

Zed developers work on Zed codebase mainly, and currently do not have any work planned on anything related to graphics backends.
